src/main/java/com/example/compound/entities/User.java

```diff
@@ -57,4 +64,6 @@ public String toString() {
     public void addExpense(Expense E) {
         this.expenses.add(E.getEUID());
     }
+
+    // TODO: removeExpense?
```

Comment:
For now let us leave it. We need to further discuss how we will redesign "group expenses".

Reply:
OK. I'll leave it as a TODO for now.